### PR TITLE
Use Redhat contact information for Quay.io

### PR DESCRIPTION
**Pull request by kurtismullins**
### Description of Changes

Remove old terms and privacy statements. Update contact information.

**NOTE**: Do **NOT** deploy this before March 1, 2020.

#### Changes:

* Remove deprecated Terms, Privacy, and Contact pages from source code.
* Update contact link in multiple pages to use the same link as shown in the footer.

#### Issue:

* https://issues.redhat.com/browse/PROJQUAY-160
* https://issues.redhat.com/browse/PROJQUAY-161
* https://issues.redhat.com/browse/PROJQUAY-162

**TESTING**

* Ensure links use the correct URL as mentioned in the changes/description and associated Jira tickets in the footer.
* Ensure the correct links are used to contact support in all changed files.
* Ensure attempting to access `/contact` `/privacy` and `/tos` results in a 404 or otherwise does not display the deprecated pages.

**BREAKING CHANGE**

n/a

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

**Comment by kurtismullins** [comment]
The footer link changes will be performed in https://github.com/quay/quay/pull/242

The contact-related changes and deleting the existing pages will be performed later once we've had a chance to discuss the changes with @josephschorr.

File: templates/base.html [hunk 1]
```diff
@@ -163,12 +163,12 @@
           <li><a href="{{ config_set['BRANDING']['footer_url'] }}" ng-safenewtab><img src="{{ config_set['BRANDING']['footer_img'] }}"></a></li>
           {% endif %}
           <li quay-require="['BILLING']"><a href="https://docs.quay.io" ng-safenewtab>Documentation</a></li>
-          <li quay-require="['BILLING']"><a href="/tos" target="_self">Terms</a></li>
-          <li quay-require="['BILLING']"><a href="/privacy" target="_self">Privacy</a></li>
+          <li quay-require="['BILLING']"><a href="https://www.openshift.com/legal/terms">Terms</a></li>
+          <li quay-require="['BILLING']"><a href="https://www.redhat.com/en/about/privacy-policy">Privacy</a></li>
```
Comment (kurtismullins) [inline templates/base.html]: #242 fixes these correctly. Ensure that this PR doesn't modify the target after #242 is merged into master.

**Comment by kurtismullins** [comment]
As the `/contact` page is not going to be removed, I am closing this PR and will be opening a new one.

**Comment by kurtismullins** [comment]
For historical purposes, the PR which replaced this one is available here: https://github.com/quay/quay/pull/262
